src/main/java/uk/gov/hmcts/reform/sscs/model/Dow.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+@Getter
+public enum Dow {
+
+    MONDAY("Monday"),
+    TUESDAY("Tuesday"),
+    WEDNESDAY("Wednesday"),
+    THURSDAY("Thursday"),
+    FRIDAY("Friday"),
+    SATURDAY("Saturday"),
+    SUNDAY("Sunday");
+
+    private final String dowLabel;
+
+    Dow(String dowLabel) {
+        this.dowLabel = dowLabel;
+    }
+
+
+    public static Dow getByLabel(String label) {
+        return Arrays.stream(Dow.values())
```

Comment:
Same comment for this method (and the rest)

If they are needed could you extract this out?

Reply:
Will change name, I just copied from confluence page and kept same name.